mixer/tools/codegen/pkg/interfacegen/generator.go

```diff
@@ -31,32 +31,34 @@ import (
 	"istio.io/istio/mixer/tools/codegen/pkg/modelgen"
 )
 
+const (
+	ResourceMsgTypeSuffix      = "Type"
+	ResourceMsgInstParamSuffix = "InstanceParam"
+	fullGoNameOfValueTypeEnum  = "istio_mixer_v1_config_descriptor.ValueType"
+	goFileImportFmt            = "\"%s\""
+	protoFileImportFmt         = "import \"%s\";"
```

Comment:
fwiw: you can use raw string literals if you would like to avoid escaping nested quotes.